src/App.js

```diff
@@ -29,37 +30,113 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
+  const [player1, togglePlayer1] = useState(true);
+  const [gameover, checkGame] = useState(false);
+
+  const togglePlayers = () => {
+    if (player1 === true) {
+      togglePlayer1(false);
+    } else {
+      togglePlayer1(true);
+    }
+  };
```

Comment:
This is a great spot to use a ternary

```suggestion
  const togglePlayers = () => {
    togglePlayer( if player1 === true ? false : true }
  };
```